common/src/api/external/mod.rs

```diff
@@ -2355,6 +2355,10 @@ pub struct SwitchPortSettingsView {
     /// Link-layer discovery protocol (LLDP) settings.
     pub link_lldp: Vec<LldpLinkConfig>,
 
+    /// TX equalization settings.  These are optional, and most links will not
+    /// need them.
```

Comment:
When will links need this setting? Is there an issue / document we can reference for this feature?

Reply:
This is needed for specific models of transceiver which don't autotune and don't work with the normal default settings.  So far, we've only found one of those, and that is now hardcoded in `dendrite`.  For now, this is for internal use only, by whoever is doing the initial customer install.